webpack.config.ts

```diff
@@ -41,6 +41,9 @@ const loaders = {
 	css: {
 		loader: 'css-loader',
```

Comment:
Is this still needed?

Reply:
Yes, because Sass compiles to CSS, so it still needs to load the CSS.